src/mesh/analysis.rs

```diff
@@ -222,9 +265,27 @@ pub fn border_edge_loops(edge_sharing: &EdgeSharingMap) -> Vec<Vec<OrientedEdge>
                     break;
                 }
 
+                let other_edge_reverted = other_edge.to_reverted();
+                if other_edge_reverted.chains_to(front) {
+                    current_chain.push_front(other_edge_reverted);
+                    border_edges.remove(i);
+                    found = true;
+
+                    break;
+                }
```

Comment:
This is a bit weird, because the reverted edge was not present in the original mesh, yet we output it in the result. Should this be the default behavior or should perhaps this function have a parameter determining whether we want this? Should we even be fixing it, and if so, should we be doing it here?

Reply:
the oriented edge (halfedge) is an internal technicality of mesh math. for a user, there is only an edge. and so is an edge loop. so if an edge is refused to become a part of a visually obvious edge loop, then it seems like an error.

from the other perspective - what good is it to be so strict about the edge orientation?

Reply:
> what good is it to be so strict about the edge orientation?

IDK, I am strict about everything by defautt :joy: , but I don't know mesh math.

> if an edge is refused to become a part of a visually obvious edge loop, then it seems like an error.

Ok, understood. half-edges are only our internal thing. Maybe let's add this to the doc comment then? Explain why it internally operates on oriented edges, but detects loops as though the edges were not oriented?

Reply:
it's an internal thing of every mesh software. but true, i'll document it.